README.md

```diff
@@ -1 +1,141 @@
-# Catalog-My-Things
+# Catalog-My-Things
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [Catalog-My-Things](#catalog-my-things)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Catalog of my things ](#-catalog-of-my-things-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+    - [Getting Started ](#getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Run](#run)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [🙏 LICENSE ](#-license-)
+
+
+# 📖 Catalog of my things <a name="about-project"></a>
+
+**Catalog of my things** In this project, you will create a console app that will help you to keep a record of different types of things you own: books, music albums, movies, and games. Everything will be based on the UML class diagram presented below. The data will be stored in JSON files but you will also prepare a database with tables structure analogical to your program's class structure.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server Side</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">RUBY</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Add Books class
+- Add MusicAlbum class
+- Add Game class
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Video Link
+
+[Video Link](https://drive.google.com/file/d/1HeqUUI60vke1nOOsYwMh-eXNlId1d4rI/view?usp=sharing)
+
+<!-- GETTING STARTED -->
+### Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Ruby inatalled on your machine
+
+### Run
+
+Example commands:
+
+```
+  ruby solver.rb
+```
+
+test 
+
+```
+  rspec solver.rb
+```

```

Comment:
- [ ] I suggest you put up the right file for starting the app, `ruby main.rb`. Running `solver` file will not start the app.